langlib/langlib-test/src/test/resources/test-src/valuelib_test.bal

```diff
@@ -1050,3 +1050,181 @@ function assert(anydata actual, anydata expected) {
         panic e;
     }
 }
+
+///////////////////////// Tests for `requireType()` ///////////////////////////
+
+json  p = {name: "Chiran", age: 24, email: "chirans", height: 178.5, weight: 72.5, property: (), address: [125.0/3, "xyz street",
```

Comment:
Shall we properly format the json as it will improve the readability?

Reply:
Fixed.